example/app/app.component.ts

```diff
@@ -7,8 +14,26 @@ import { Component, ViewEncapsulation, ViewChild } from '@angular/core';
   encapsulation: ViewEncapsulation.None,
 })
 export class AppComponent {
+  @ViewChild('myExampleChatbot') myExampleChatbot: ChatbotComponent;
+
   // start new random sessions every time
   public session1: string = Math.random().toString(36).substring(7);
   public session2: string = Math.random().toString(36).substring(7);
```

Comment:
Question unrelated to this pr but why create 2 sessions?

Reply:
Removed redundant code.